Utils/PoolableObject.cs

```diff
@@ -0,0 +1,24 @@
+﻿using DUCK.Utils;
+using System;
+using UnityEngine;
+
+namespace DUCK.Pooling
+{
+	public class PoolableObject : MonoBehaviour
```

Comment:
Minor bit of weirdness, PoolableObject inherits MonoBehaviour but this whole feature references objects which is a little further up the inheritance tree.

Reply:
I don't follow - where does the ObjectPool reference objects as object/Object?

Reply:
In the naming convention.